deploy_docker/apache2.conf

```diff
@@ -209,11 +209,7 @@ AccessFileName .htaccess
 # Note that the use of %{X-Forwarded-For}i instead of %h is not recommended.
 # Use mod_remoteip instead.
 #
-LogFormat "%v:%p %h %l %u %t \"%r\" %>s %O \"%{Referer}i\" \"%{User-Agent}i\"" vhost_combined
-LogFormat "%h %l %u %t \"%r\" %>s %O \"%{Referer}i\" \"%{User-Agent}i\"" combined
-LogFormat "%h %l %u %t \"%r\" %>s %O" common
-LogFormat "%{Referer}i -> %U" referer
-LogFormat "%{User-agent}i" agent
+
```

Comment:
I see now that you moved these out of `apache2.conf` and into `000-default.conf`.  I think the comments on lines 201 - 211 should go along with the values moved.

Reply:
Just moved those comments to `000-default.conf`